src/Razor/Razor/src/TagHelpers/HtmlAttributeValueStyle.cs

```diff
@@ -1,12 +1,30 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
 namespace Microsoft.AspNetCore.Razor.TagHelpers;
 
+/// <summary>
+/// Determines how a HTML attribute value appears in markup.
```

Comment:
```suggestion
/// Determines how an HTML attribute value appears in markup.
```

I'm not a grammar expert, but I think this should be 'an'